scmail.py

```diff
@@ -184,7 +211,7 @@ def send(ctx, recipient, message):
         logging.info('Message encrypt success.')
 
     # Send
-    payload = {'fingerprint': recipient, 'message': message}
+    payload = {'fingerprint': recipient, 'message': message, 'sender_fingerprint': sender_fingerprint}
```

Comment:
To send over the encrypted message:

```suggestion
    payload = {'fingerprint': recipient, 'message': encrypted_message.decode(), 'sender_fingerprint': sender_fingerprint}
```